### ISSUE TITLE
How much psnr and ssim do you get?

### ISSUE THREAD

**Issue by Jamescallow** [MayankSingal/VideoExpansion#1]
Hi, I want to know how much psnr and ssim you can get on Adobe-240 dataset after training? I have an implementation of tensorflow and the psnr is about 26 after training about 200 epoches, can't get 30+ psnr as the paper said.

**Comment by MayankSingal** [comment]
Hey @XingJun940207 
Training on Adobe-240 gives around 26 psnr after 200 epochs for me too. I haven't had a chance to run it for longer due to constrained resources. Adding the NFS dataset [http://ci2cv.net/nfs/index.html](url), got the psnr up to ~27.
Cheers!

**Comment by hogeman2** [comment]
I also tried to reproduce papers results. I got similar psnr 26~27 around 200 epoches. 
maybe some extra technique is needed to get better performance(augmentation or something).

**Comment by youyingyin** [comment]
> I also tried to reproduce papers results. I got similar psnr 26~27 around 200 epoches.
> maybe some extra technique is needed to get better performance(augmentation or something).

may be using different Perceptual loss? The origianl paper used vgg16 model, while this code used resnet18
